src/import/import-figma.ts

```diff
@@ -348,8 +369,11 @@ export const getFontFamilies = (
   // Convert array to object
   const fontMap = fonts.reduce(
     (obj, s) => ({
+      $type: 'fontFamily',
       ...obj,
-      [s.name]: s.fontFamily,
+      [s.name]: {
+        $value: `'${s.fontFamily}'`,
```

Comment:
I'm curious why you moved the wrapping quotes to this stage of the process rather than at the point of rendering it to a CSS property, is this something mandated by the Style Dictionary format?

Reply:
I moved it as there aren't any situations (that I can think of) where we need the font family without the quotes around it, so this way we don't have to go in for each exporter and make sure to add the quotes.
I'm not heart bound to this though, have to change it if you've got a reason to.